jdaviz/configs/default/plugins/collapse/collapse.py

```diff
@@ -47,16 +83,22 @@ def vue_collapse(self, *args, **kwargs):
             warnings.filterwarnings('ignore', message='No observer defined on WCS')
             spec = spectral_slab(cube, spec_min, spec_max)
             # Spatial-spatial image only.
-            collapsed_spec = spec.collapse(self.selected_func.lower(), axis=-1).T  # Quantity
+            collapsed_spec = spec.collapse(self.method_selected.lower(), axis=-1).T  # Quantity
+
+        if add_data:
+            data = Data()
+            data['flux'] = collapsed_spec.value
+            data.get_component('flux').units = str(collapsed_spec.unit)
 
-        data = Data()
-        data['flux'] = collapsed_spec.value
-        data.get_component('flux').units = str(collapsed_spec.unit)
+            self.add_results.add_results_from_plugin(data)
 
-        self.add_results.add_results_from_plugin(data)
+            snackbar_message = SnackbarMessage(
+                f"Data set '{self.dataset_selected}' collapsed successfully.",
+                color="success",
+                sender=self)
```

Comment:
I'm not seeing this snackbar message either when I use the API or the UI. Do you see it trigger successfully?

Reply:
I think success messages are filtered to the logger window by default.  Is it there?

Reply:
Ahhh right, I forgot we had filtered the snackbar, it's there. I wonder if it would be useful to add an indicator to the logger icon if there are unviewed messages there...